stubs/Stringable.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+if (\PHP_VERSION_ID < 80000) {
```

Comment:
How about another if statement to make sure the interface does not exist already, to ensure compatibility with other packages doing the same thing.

Reply:
✅